modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -4,77 +4,203 @@ import {Meta} from '@storybook/addon-docs';
 
 # Canvas Kit 8.0 Upgrade Guide
 
-This guide contains breaking changes in Canvas Kit v8. Please
+This guide contains an overview of the changes in Canvas Kit v8. Please
 [reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you have
 any questions.
 
 - [Codemod](#codemod)
-- [Components](#components)
-  - [Component Deprecations](#component-deprecations)
+- [General Updates](#general-updates)
+  - [IE11 No Longer Supported](#ie11-no-longer-supported)
+  - [Removal of Default Exports](#removal-of-default-exports)
+  - [Responsive Styles](#responsive-styles)
+  - [Refactor Style Props](#refactor-style-props)
+  - [Testing](#testing)
+- [New Components](#new-components)
+  - [Grid](#grid)
+  - [Text](#text)
+  - [Status Indicator (Preview)](#status-indicator-preview)
+- [Component Deprecations](#component-deprecations)
+  - [Drawer](#drawer)
+  - [Menu (Preview)](#menu-preview)
+  - [Layout and Column](#layout-and-column)
+- [Component Updates](#component-updates)
   - [Box](#box)
-  - [Menu (Main)](#menu)
   - [Breadcrumbs](#breadcrumbs)
-- [General Changes](#general-changes) -[Remove Default Imports](#remove-default-imports)
+  - [Menu (Main)](#menu-main)
+  - [Modal](#modal)
+  - [Segmented Control](#segmented-control)
```

Comment:
This should be moved to the `New Components` since it's a net-new component being added to Preview (not affecting the Segmented Control currently in Main).